html/pfappserver/root/src/views/Auditing/_router/index.js

```diff
@@ -1,179 +1,27 @@
 import acl from '@/utils/acl'
-import i18n from '@/utils/locale'
-import store from '@/store'
-import RadiusLogsStore from '../_store/radiusLogs'
-import DhcpOption82LogsStore from '../_store/dhcpOption82Logs'
-import DnsLogsStore from '../_store/dnsLogs'
-import AdminApiAuditLogs from '../_store/adminApiAuditLogs'
-import LiveLogs from '../_store/liveLogs'
+import AdminApiLogsRoutes from '../adminApiLogs/_router'
+import DhcpOption82LogsRoutes from '../dhcpOption82Logs/_router'
+import DnsLogsRoutes from '../dnsLogs/_router'
+import RadiusLogsRoutes from '../radiusLogs/_router'
+import LiveLogsRoutes from '../liveLogs/_router'
```

Comment:
This file is missing.

Reply:
Missing file added in a4f46426f4a23dbca9ff28c0e109ee1bc541c435